features/importers/venepaikka_harbors/importer.py

```diff
@@ -32,6 +32,29 @@
           phone
           email
           servicemapId
+          maximumWidth
+          maximumLength
+          maximumDepth
```

Comment:
these will be renames to `max<Width|Length|Depth>` in the next couple of days. just as a heads up 😅